featuretools/primitives/utils.py

```diff
@@ -385,6 +387,10 @@ def deserialize_primitive(self, primitive_dict):
                 'Primitive "%s" in module "%s" not found' % (class_name, module_name),
             )
         arguments = primitive_dict["arguments"]
+        if cls.__name__ == "NumberOfCommonWords" and "word_set" in arguments:
```

Comment:
cant you compare class objects directly?

Reply:
fixed!